localstack/utils/bootstrap.py

```diff
@@ -213,7 +214,13 @@ def get_enabled_apis() -> Set[str]:
 
     The result is cached, so it's safe to call. Clear the cache with get_enabled_apis.cache_clear().
     """
-    return resolve_apis(config.parse_service_ports().keys())
+    services = os.environ.get("SERVICES", "").strip()
```

Comment:
I guess we still need to split the string here? (similar to what we had in `parse_service_ports()` before)

Reply:
Good point. We would not support the different service ports anyway, but I re-added the parsing (but the ports will just be discarded).